docs/posts/bigquery-arrays/index.qmd

```diff
@@ -0,0 +1,231 @@
+# Working with arrays in Google BigQuery
```

Comment:
can you add the frontmatter? example from v6.1.0 release blog:

```
---
title: Ibis v6.1.0
author: "Ibis team"
date: "2023-08-02"
categories:
    - release
    - blog
---
```

Reply:
Yep, thanks!